parts/linux/cloud-init/artifacts/cis.sh

```diff
@@ -124,10 +124,22 @@ setPWExpiration() {
     replaceOrAppendUserAdd INACTIVE 30
 }
 
+function fixRpFilter() {
+    # Mariner AKS CIS Benchmark: 3.3.7 Ensure Reverse Path Filtering is enabled
+    # Comment out rp_filter settings in /usr/lib/sysctl.d/50-default.conf and /lib/sysctl.d/50-default.conf
+    # We set these correctly in sysctl-d-60-CIS.conf, but they also need to not be present in the default file.
+    # These sed comands will comment out any line that starts with the settings we want to get rid of.
+    sed -E -i 's/^[[:space:]]*net.ipv4.conf.all.rp_filter/#net.ipv4.conf.all.rp_filter/g' /usr/lib/sysctl.d/50-default.conf
+    sed -E -i 's/^[[:space:]]*net.ipv4.conf.default.rp_filter/#net.ipv4.conf.default.rp_filter/g' /usr/lib/sysctl.d/50-default.conf
+    sed -E -i 's/^[[:space:]]*net.ipv4.conf.all.rp_filter/#net.ipv4.conf.all.rp_filter/g' /lib/sysctl.d/50-default.conf
+    sed -E -i 's/^[[:space:]]*net.ipv4.conf.default.rp_filter/#net.ipv4.conf.default.rp_filter/g' /lib/sysctl.d/50-default.conf
```

Comment:
This is going to cause CIS scanning failures if systemd has been updated on the system. Our systemd package doesn't tag /usr/lib/sysctl.d/50-default.conf as a %config(noreplace) [though it probably should], so any upgrade to systemd is going to stomp the file entirely.